test/runcitests.cmd

```diff
@@ -127,10 +130,13 @@ set _HadFailures=0
 
   call :do %_TestDir%\runnativetests.cmd -%1%2 > %_TestDir%\logs\%1_%2\nativetests.log 2>&1
 
-  if ERRORLEVEL 1 set _HadFailures=1
+  if %ERRORLEVEL% NEQ 0 (
+    echo -- runcitests.cmd ^>^> runnativetests.cmd failed

```

Comment:
entire log would not be necessary, however point to the location of the log would be good help. 

---
In reply to: [88581337](https://github.com/Microsoft/ChakraCore/pull/2043#discussion_r88581337) [](ancestors = 88581337)